golang.sh

```diff
@@ -0,0 +1,47 @@
+package: golang
+version: "1.10.2"
+build_requires:
+  - curl
+prefer_system_check: |
+  which go && case `go version | sed -e 's/go version go//' | sed -e 's/ .*//'` in 0*|1.[0-9].*) exit 1 ;; esac
+incremental_recipe: |
```

Comment:
This incremental recipe is useless, please drop it completely (it does precisely the same as the full one).